apps/examples/testcase/le_tc/network/tc_net_getpeername.c

```diff
@@ -125,12 +124,13 @@ static void tc_net_getpeername_close_n(void)
 	sock = socket(AF_INET, SOCK_STREAM, 0);
 	int len = sizeof(struct sockaddr);
 	struct sockaddr foo;
-	close(sock);
+
 	int ret = getpeername(sock, &foo, (socklen_t *)&len);
 
 	TC_ASSERT_EQ("getpeername", ret, -1);
 	TC_SUCCESS_RESULT();
-	
+
+	close(sock);
```

Comment:
if you close here, handle 'sock' can be lost when the fail at TC_ASSERT_EQ("getpeername", ret, -1);